### ISSUE TITLE
EDS clustering demo (notebook)

### ISSUE THREAD

**Issue by jorgenasorhaug** [hyperspy/exspy-demos#5]
I recently published a paper in Materials & Design where I employed cluster analysis (k-means) to cluster EDS spectra from a bimetallic weld. The objective was to highlight the presence of segregated elements within intermetallic phase layers. Is there of interest to make a notebook where I show this?

See: https://doi.org/10.1016/j.matdes.2024.112867

**Comment by ericpre** [comment]
Yes, that would be useful, providing that the notebook explains what it does and the reasoning behind the various steps.

**Comment by jorgenasorhaug** [comment]
Great. I'll make one as soon as time allows :)
